docs/proposals/network-policy.md

```diff
@@ -197,6 +254,35 @@ type NetworkPolicyPort struct {
 	Port *intstr.IntOrString `json:"port,omitempty"`
 }
 
+type NetworkCIDRRange struct {
+	// QUESTION: we could just do "Start" and "End" without a prefix to
+	// specify the range.  While this provides a simpler structure, it may
+	// be more error-prone since its less easily validated, and isn't very
+	// nice for IPv6 addresses.
```

Comment:
> Possibly simpler representation:
> 
> Start: 0.0.0.0, Prefix: 8, End: 9.0.0.0
> Start: 11.0.0.0, Prefix: 24, End: 192.168.41.0
> Start: 192.168.43.0, Prefix: 24, End: 255.255.255.0

That seems like it shouldn't validate... eg, "0.0.0.0/8" means "0.0.0.0 - 0.255.255.255", so End is not in range... unless Prefix is supposed to apply to End but not Start in that case.

My example was wrong too though... I should have used Prefix: 0, not Prefix: 32. So... both of us managed to get it wrong here, plus your example in your email had a typo/thinko in it, plus one of the two examples in the patch is wrong. So I'm thinking that we absolutely don't want to allow both Prefix and End together.